lib/xcode/install.rb

```diff
@@ -59,7 +59,7 @@ def fetch(url: nil,
       progress_log_file = File.join(CACHE_DIR, "progress.#{Time.now.to_i}.progress")
       FileUtils.rm_f(progress_log_file)
 
-      retry_options = ['--retry', '3']
```

Comment:
I've never noticed such variable, is this working?

Reply:
what I checked is below

### remove `retry_options = ['--retry', '3']` and `--retry-download-count=2`

```shell
% bundle exec bin/xcversion install 12\ beta\ 2  --retry-download-count=2
Available session is not valid any more. Continuing with normal login.
100   305  100   305    0     0    856      0 --:--:-- --:--:-- --:--:--   856
100   305  100   305    0     0    856      0 --:--:-- --:--:-- --:--:--   856
  7 9040M    7  659M    0     0  4554k      0  0:33:52  0:02:28  0:31:24 9239k%
curl: (18) transfer closed with 8788212621 bytes remaining to read
100   319  100   319    0     0    523      0 --:--:-- --:--:-- --:--:--   522% # continued twice even if remove `retry_options = ['--retry', '3']`
```

curl continued twice even if remove `retry_options = ['--retry', '3']` 

### leave `retry_options = ['--retry', '3']` and `--retry-download-count=1`

```shell
% bundle exec bin/xcversion install 12\ beta\ 2  --retry-download-count=1
Available session is not valid any more. Continuing with normal login.
100   305  100   305    0     0    381      0 --:--:-- --:--:-- --:--:--   381%
 21 9040M   21 1920M    0     0  5639k      0  0:27:21  0:05:48  0:21:33     0%
curl: (18) transfer closed with 7465630605 bytes remaining to read
%[!] Failed to download Xcode 12 beta 2. # stopped even if `retry_options = ['--retry', '3']` is still in code 
```

curl stopped even if `retry_options = ['--retry', '3']` is still in code

Reply:
so I think setting `retry_options` can be removed, what do you think?

Reply:
when we remove it, nothing changed. It is passed as parameters to the `Open3.popen3()`, I checked the [doc of popen3](https://docs.ruby-lang.org/en/2.7.0/Open3.html#method-i-popen3 ) I do not see anything tied to `retry` so we can delete it.

Reply:
I think we still want to keep this in 🤔 

```
--retry <num>   Retry request if transient problems occur
```

There was a whole thing about this 3 years ago - https://github.com/xcpretty/xcode-install/pull/217

cc: @marcomorain This is a thing we still want to keep in here, right?

Reply:
looks #217 was done for the same motivation (curl 18 error) and it now not works well, as I wrote above. So I think we can remove it.